ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -365,7 +365,7 @@ select_ofi_provider(struct fi_info *providers,
     if (NULL != prov) {
         prov = opal_mca_common_ofi_select_provider(prov, &ompi_process_info);
         opal_output_verbose(1, ompi_mtl_base_framework.framework_output,
```

Comment:
while you're here, one thing that caused some confusion recently in AWS (and was on my todo list to clean up) is that most of the selection output goes to `opal_common_ofi.output`, but this one goes to `ompi_mtl_base_framework.framework_output`, which means the verbose flags are different.

Reply:
@bwbarrett Do you want all selection output to go to `opal_common_ofi.output`? If so, I can do that, too.

Reply:
I think @hppritcha was the one who moved most of it to opal_common_ofi.output.  I don't have strong opinions.  In some ways, it does seem like it's an easier discussion with users to set one flag and get both the OFI MTL and BTL output, so I lean towards opal_common_ofi.  But mainly it's confusing why some prints "work" and others "don't work" when you only set one of the two verbose MCA parameters :).

Reply:
Thanks, @bwbarrett. db837b3 normalizes `opal_output_verbose()` sinks.